app/customer.py

```diff
@@ -0,0 +1,45 @@
+import math
+
+from app.car import GasStation, Car
+from app.shop import Shop
+
+
+class Customer:
+    fuel_price = GasStation(2.4)
```

Comment:
why you save fuel_price in Customer
Firstly we take fuel_price from json file
Secondly how fuel_price connect with customer?

Reply:
Also i don't think we need class for this, but maybe ok